README.md

```diff
@@ -80,6 +80,40 @@ You can download and install range-v3 using the [vcpkg](https://github.com/Micro
 
 The range-v3 port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
 
+Building range-v3 - Using Conan
+-------------------------------
+
+You can download and install range-v3 using the [Conan](https://github.com/conan-io/conan) dependency manager.
+
+Setup your CMakeLists.txt (see [Conan documentation](https://docs.conan.io/en/latest/integrations/build_system/cmake.html) on how to use `find_package` with CMake instead, and how to use different [buid systems](https://docs.conan.io/en/latest/integrations/build_system.html) too):
```

Comment:
This parenthetical here seems to confuse rather than clarify. Are users to set up their CMakeLists.txt as you show below? Or are they to do something different using `find_package`? Why should they prefer one over the other? 

The reference to different build systems further confuses me. CMake isn't really a build system, it's a generator. If you're going to mention build systems at all, can you be clearer about how it relates to the discussion of CMake and Conan?

Reply:
>  CMake isn't really a build system, it's a generator.

While technically this is correct, I believe it's too nit-picky to not call CMake a build system. How else do we call all similar software that allows to declare how your project is being built? Conan documentation uses "build systems" tern in the link provided too, bringing CMake, Make, and others together.

Anyhow, I will simplify parentheses. All that info is probably off-topic for a quick start example. I wanted for better but.. :-) .